pkg/build/pipelines/ruby/require.yaml

```diff
@@ -0,0 +1,85 @@
+name: Test a Ruby package require, with optional load clause
+
+needs:
+  packages:
+    - wolfi-base
+
+inputs:
+  ruby:
+    description: Which Ruby to use
+    default: DEFAULT
+  require:
+    description: |
+      The package to require.
+    required: false
+  requires:
+    description: |
+      Commands to require packages, each line is a separate command. Example:
+        require 'foo'
+        # test that otherthing can be required from asdf
+        require 'asdf'
+        require 'bark' # this is like woof
+
+      full-line and inline comments are supported via '#'
+    required: false
+
+pipeline:
+  - runs: |
+      set +x
```

Comment:
I don't think this applies to the rest of the pipeline

Reply:
This is a copy-paste thing from `python/import` pipeline actually. I'm checking the CI logs [here](https://github.com/chainguard-dev/melange/actions/runs/10778428673/job/29889753721?pr=1483#step:9:1075) but couldn't see the instructions either.

Reply:
Yeah, happy to be wrong here, but I don't believe it's transient to the rest of the pipeline. Will let someone that knows for sure chime in

Reply:
it does "apply", in that it turns off '-x' option (making it not spew bash debug to stderr).